src/store/selectors/utils.js

```diff
@@ -0,0 +1,8 @@
+import { mapValues } from '../../utils/func.js'
+import { createSelector } from 'reselect'
+
+export const createSelectorWhichMapsToHash = (hashSelector, mapFunc) =>
+  createSelector(hashSelector, capturesValue => mapValues(capturesValue, mapFunc))
+
+export const createValuesHashToValueSelector = (valuesHashSelector, valueKey) =>
+        createSelector(valuesHashSelector, valuesHash => valuesHash[valueKey])
```

Comment:
What's with the really inconsistent indentation? Seems to be in a few places around this PR